src/leap/keymanager/validation.py

```diff
@@ -24,17 +24,17 @@
 
 
 from datetime import datetime
-from enum import Enum
+from enum import IntEnum
 
 
-ValidationLevel = Enum(
-    "Weak_Chain",
-    "Provider_Trust",
-    "Provider_Endorsement",
-    "Third_Party_Endorsement",
-    "Third_Party_Consensus",
-    "Historically_Auditing",
-    "Known_Key",
+ValidationLevel = IntEnum("ValidationLevel",
+    "Weak_Chain "
+    "Provider_Trust "
+    "Provider_Endorsement "
+    "Third_Party_Endorsement "
+    "Third_Party_Consensus "
+    "Historically_Auditing "
+    "Known_Key "
```

Comment:
I migrated it to enum34, where the values of the enum are not separated with _,_ as in enum but they are one string with the values separated with spaces. I think is easier to read it if I keep the string like that changing lines for each value, but it's not pep8 compliant.